projects/samples/devices/worlds/console.wbt

```diff
@@ -0,0 +1,33 @@
+#VRML_SIM R2020a utf8
```

Comment:
To me this sample as it is not strictly a device, should rather go in ``projects/samples/howto`` (as it is the case for example for the sample showing how to use the mouse (which is not a Webots device too)).